deposit/src/main/java/edu/unc/lib/deposit/normalize/AbstractMETS2N3BagJob.java

```diff
@@ -99,7 +99,7 @@ protected void assignPIDs(Document mets) {
 				div.setAttribute("CONTENTIDS", sb.toString());
 			}
 			
-			PremisLogger premisLogger = getPremisLogger(pid);
```

Comment:
I would suggest changing the behavior of `getPremisLogger` to call `premisLoggerFactory.createPremisLogger` rather than replacing its usage.  Doing so would update all of our deposit jobs.  Additionally, `getPremisLogger` actually calls `new FilePremisLogger(pid, file, repository);`, passing in the file is pretty critical in this case